### PR TITLE
Add test for torch models tracing

### PR DESCRIPTION
**Pull request by lostella**
*Description of changes:* Add test to check that tracing a model results in the same computation, i.e., that the model's behaviour does not depend on input values.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: test/torch/model/test_jit.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from gluonts.torch.model.deepar import DeepARModel
+from gluonts.torch.model.tft import TemporalFusionTransformerModel
+
+
+def get_model_and_input(model):
+    return model, tuple(
+        [
+            torch.zeros(shape, dtype=model.input_types()[name])
+            for (name, shape) in model.input_shapes().items()
```
Comment (jaheba) [inline test/torch/model/test_jit.py]: I think it would be nice to not have two different methods to get information on values.

Comment (lostella) [reply test/torch/model/test_jit.py]: OK, I guess that can be addressed separately?

Comment (jaheba) [reply test/torch/model/test_jit.py]: Yes!

File: test/torch/model/test_jit.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+import pytest
+
+import torch
+
+from gluonts.torch.model.deepar import DeepARModel
+from gluonts.torch.model.tft import TemporalFusionTransformerModel
+
+
+def get_model_and_input(model):
```
Comment (jaheba) [inline test/torch/model/test_jit.py]: What's the purpose of this, why not just use `.example_input_array`?

Also, why should this be part of the parameter and not just be invoked in the test function?

**Review by jaheba — COMMENTED** [review]
There also is `to_torchscript` which lightning provides.

File: test/torch/model/test_jit.py (hunk 2 above)
Comment (lostella) [reply test/torch/model/test_jit.py]: I'm not sure that using these completely dummy arrays is the best approach. Maybe for tracing the model yes, but then we should probably test the model vs scripted model using some other input

Comment (lostella) [reply test/torch/model/test_jit.py]: So it's mostly a placeholder for now

Comment (lostella) [reply test/torch/model/test_jit.py]: > What's the purpose of this, why not just use .example_input_array?

That's not a property of `torch.nn.Module`

Comment (jaheba) [reply test/torch/model/test_jit.py]: Right. Still looks like something which could be consolidated.

**Review by jaheba — APPROVED** [review]
